README.md

```diff
@@ -1,2 +1,141 @@
-# Mobile-clinic
-Repository for the website of a mobile medical clinic. This innovative approach to healthcare delivery brings medical professionals and resources directly to underserved or remote areas, ensuring that people have access to essential medical services even if they are unable to reach a fixed healthcare facility.
+<a name="readme-top"></a>
+
+<div align="center">
+  <h2>Filbert Portfolio</h2>
+  <br/>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
```

Comment:
- [ ] Kindly deploy your website as per project requirements and attach the live demo link in the README file. 🙏 check it [here](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=link%2C%20mobile%20menu.-,Deploy%20the%20project.,-Day%203)